### PR TITLE
 Fix the ci tagging script + update `che.version` to `5.20.0-SNAPSHOT`

### PR DESCRIPTION
**Pull request by davidfestal**
### What does this PR do?

Fix the ci tagging script + update `che.version` to `5.20.0-SNAPSHOT`

### CODE REVIEW

File: .ci/cico_do_docker_build_tag_push.sh [hunk 1]
```diff
@@ -60,9 +60,10 @@ do
   docker rmi eclipse/che-server:local
 
   # lets change the tag and push it to the registry
-  docker tag ${DOCKER_HUB_NAMESPACE}/che-server:${NIGHTLY} ${DOCKER_HUB_NAMESPACE}/che-server:${TAG}
-  
-  dockerTags="${dockerTags} ${DOCKER_HUB_NAMESPACE}/che-server:${NIGHTLY} ${DOCKER_HUB_NAMESPACE}/che-server:${TAG}"
+docker tag ${DOCKER_HUB_NAMESPACE}/che-server:${NIGHTLY} ${DOCKER_HUB_NAMESPACE}/che-server-multiuser:${TAG}
```
Comment (l0rd) [inline .ci/cico_do_docker_build_tag_push.sh]: Bad indentation

File: pom.xml [hunk 2]
```diff
@@ -16,7 +16,7 @@
     <parent>
         <artifactId>che-parent</artifactId>
         <groupId>org.eclipse.che</groupId>
-        <version>5.19.0-SNAPSHOT</version>
+        <version>5.20.0-SNAPSHOT</version>
```
Comment (l0rd) [inline pom.xml]: You need to update https://github.com/redhat-developer/rh-che/blob/multi-tenant-rh-che/config#L23 too

Comment (davidfestal) [reply pom.xml]: Fixed

File: config [hunk 3]
```diff
@@ -20,4 +20,4 @@ export DOCKER_HUB_USER=${DOCKER_HUB_USER:-${RHCHEBOT_DOCKER_HUB_USER}}
 export DOCKER_HUB_PASSWORD=${DOCKER_HUB_PASSWORD:-${RHCHEBOT_DOCKER_HUB_PASSWORD}}
 export DOCKER_HUB_NAMESPACE=${DOCKER_HUB_NAMESPACE:-${RH_CHE_DOCKER_HUB_NAMESPACE}}
 
-export CHE_DOCKER_BASE_IMAGE=eclipse/che-server:5.18.0-centos
+export CHE_DOCKER_BASE_IMAGE=eclipse/che-server-multiuser:5.20.0-centos
```
Comment (l0rd) [inline config]: I don't this tag exist yet. You should use nightly-centos instead

**Pull request merged**